### ISSUE TITLE
colormaps not preserved in session files for derived variables

### ISSUE THREAD

**Issue by clyne** [NCAR/VAPOR#1763]
### Describe the bug
If a colormap is defined (edited) for a derived variable it will not be preserved from a session file. Actually, it looks like the color map may be preserved, but it is not restorable. Additionally, a derived variable stored in a session file will not be accessible to the application until it is explicitly reloaded via the Python Tools editor.

**Helpful additional information**
(*Please click check boxes AFTER submitting ticket*)
- [ ] Did VAPOR crash?
- [x] Did you get wrong results?

**Impact**
- [ ] High - User productivity significantly degraded
- [x] Medium - User productivity partially degraded
- [ ] Low - User productivity trivially degraded

### To Reproduce
Steps to reproduce the behavior. For example:
1. Load Duku 
2. Define a derived variable 'windspeed' via the Python tools editor:
```
import numpy as np
windspeed = np.sqrt(U10*U10 + V10*V10)
```
Be sure to select U10 and V10 as input variables and define windspeed as an output variable. Also, be sure to click "save" in the Python editor
3. Enable the 2D renderer with the windspeed variable and modify the color map
4. Save the session to a file and exit
5. Restart vapor, and load just saved session file. The application should start with the 2D renderer enabled and windspeed the selected variable. It does not and may generate an error that windspeed is not found. 
6. Go to the python editor and select "open from session" to reload the script for windspeed.
7. Selected windspeed from the 2D renderer. The variable will now be displayed but it will use the default color map, not the one saved in the session.

**Comment by clyne** [comment]
re-opened. This bug is back

**Comment by clyne** [comment]
@stasj I'm handing this over to you. The problem a regression you introduced in commit 
6f5a51f4636b64db8bcfd841ffa2c0cfbd147612

With the change you made to ParamsBase::GetValueString() it is no longer possible to retrieve a multi-line string. In this particular case we're storing a Python script and only the first line of the script is being retrieved by ParamsBase::GetValueString() inside of VAPoR::DatasetParams::ScriptParams::GetScript().

Note, there is a remote branch already, "issue_1763". The changes in this branch may or may not be necessary. 

Note also that we should confirm correct behavior by saving and restoring a multi-line python script that contains indentation.

**Comment by StasJ** [comment]
Ah. My efforts were so that params would no longer split single values into vectors so that change would work but I didn't consider that newlines are used as a delimiter so the multiline string is interpreted as a vector in params. This implementation is rather finicky. I went ahead and reversed the change and did some testing to make sure it didn't introduce additional regressions and it seems to be working. I created a PR. I'll see if it fixes this problem too.
